include/open62541/server_pubsub.h

```diff
@@ -923,6 +924,36 @@ UA_Server_addSecurityGroup(UA_Server *server, UA_NodeId securityGroupFolderNodeI
 UA_StatusCode UA_EXPORT UA_THREADSAFE
 UA_Server_removeSecurityGroup(UA_Server *server, const UA_NodeId securityGroup);
 
+/**
+ * @brief This user defined callback to notify the user about the current status of the periodic SKS Pull request.
+ * @param server instance used to make sks pull request
+ * @param sksClientStatus
+ */
+typedef void
+(*UA_Server_sksPullRequestCallback)(UA_Server *server, UA_StatusCode sksPullRequestStatus, UA_Boolean sksPullSuccess, void* userData);
```

Comment:
The success argument is not described in the comments.
Would it be enough to have status==good the same as success=true?
Then we don’t need that argument.